src/index.d.ts

```diff
@@ -1230,9 +1230,9 @@ declare module 'react-native-firebase' {
         /**
          * Returns the devices FCM token.
          */
-        getToken(): Promise<string>;
+        getToken(senderId?: string): Promise<string>;
 
-        deleteToken(authorizedEntity?: string, scope?: string): Promise<void>;
```

Comment:
if this part didn't change the typescript method signature, this wouldn't be breaking. But I'm not sure how this can actually be valid as a type change, it seems like we're losing an API completely (that I would assume is necessary?) in order to get a new one that can take a senderId? I am probably missing something, but I need to understand

Reply:
We can make this non-breaking by keeping matching the JS code in each method to how iid() does it, with default arg values to be the same as these: 

https://github.com/invertase/react-native-firebase/blob/v5.x.x/src/modules/iid/index.js#L51-L52

https://github.com/invertase/react-native-firebase/blob/v5.x.x/src/modules/iid/index.js#L65-66

That way the old method signature with no args still continues to work as it did before.